src/PDPVerifier.sol

```diff
@@ -26,6 +26,7 @@ contract PDPVerifier is Initializable, UUPSUpgradeable, OwnableUpgradeable {
     uint256 public constant MAX_ROOT_SIZE = 1 << 50;
     uint256 public constant MAX_ENQUEUED_REMOVALS = 2000;
     address public constant RANDOMNESS_PRECOMPILE = 0xfE00000000000000000000000000000000000006;
+    uint256 public constant EXTRA_DATA_MAX_SIZE = 512;
```

Comment:
What's the justification for this number? Again, perhaps documents it. It seems rather limiting – what could go wrong if we chose a larger number?

Reply:
We could go bigger for sure, let's double it.  My thinking was 16 words are so much that its hard to think of protocols that are going to restricted by it but I think we get a little closer to the limits of PSD deal proposal so bigger is probably better.